bigquery-storage/api/BigQueryStorage.Samples.Tests/QuickstartTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(BigQueryStorageFixture))]
+public class QuickstartTest
```

Comment:
Just an observation. The existing test class in the project has a suffix `Tests`. We may want to keep it consistent?

Reply:
Yep, probably a good idea.